engines/standard_tasks/spec/models/paper_editor_task_spec.rb

```diff
@@ -2,70 +2,32 @@
 
 describe StandardTasks::PaperEditorTask do
   let(:paper) { FactoryGirl.create :paper, :with_tasks }
-  describe "#paper_role" do
-    let(:user) { FactoryGirl.build(:user) }
-    let!(:paper_role) { create(:paper_role, :editor, paper: paper, user: user) }
-    let!(:phase) { paper.phases.first }
-    let(:task) { StandardTasks::PaperEditorTask.create!(phase: phase, title: "Assign Editor", role: 'admin') }
 
-    context "when the role is not present" do
-      it "initializes a new editor role" do
-        role = task.paper_role
-        expect(role.paper).to eq paper
-        expect(role.role).to eq('editor')
-      end
-    end
+  describe "#invitation_accepted" do
 
-    context "when the role is present" do
-      it "returns the role" do
-        # Setting up associations with several has_throughs is difficulte to verify
-        # directly.
-        # After reloading the task can get its paper_role directly,
-        # but if the record hasn't been persisted we'd need to use
-        # task.phase.paper.paper_roles.where(editor: true)
-        expect(StandardTasks::PaperEditorTask.create!(phase: phase, title: "Assign Editor", role: 'admin').reload.paper_role).to eq paper_role
-      end
+    let!(:task) do
+      StandardTasks::PaperEditorTask.create!({
+        phase: paper.phases.first,
+        title: "Assign Editor",
+        role: 'admin'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.